### DISCUSSION TITLE
There are suggestions for improving accessibility.

### DISCUSSION THREAD

**Discussion by ryusei-48** [electerm/electerm#2734] [Ideas]
I am visually impaired and use a screen reader for my normal PC work.
If you are using the "xTerm.js" module for this project, could you please enable "screenReaderMode"?
Enabling this feature will allow NVDA to read screen text.

We share the documentation of the relevant part of "xTerm.js" below.
https://github.com/xtermjs/xterm.js/blob/19c760a20470c26666efad770fe82496c4a9bf1e/typings/xterm.d.ts#L192

**Comment by zxdong262** [comment]
Released 1.25.12, should support NVDA on Windows now, need turn it on in setting page first, as I tested, not as smooth as normal browser text content, may need stop at the terminal text for a little while to trigger reading.

**Comment by ryusei-48** [comment]
Thank you very much. I appreciate the quick response. I will try it as soon as possible and comment again later.

**Reply by ryusei-48** [reply]
Sorry for the very late comment, it is working fine in NVDA browsing mode.
I was previously attempting to develop a highly functional terminal app in Electron that is optimized for screen readers.
However, I recently found this great app and no longer need to develop it on my own.
I may also make accessibility requests from time to time, and I would appreciate it if you could address them if possible.

**Reply by zxdong262** [reply]
@ryusei-48 Glad it works fine, thank you for the feedback, I may never know this without your suggestions, feel free to let me know any idea/suggestion/request.
